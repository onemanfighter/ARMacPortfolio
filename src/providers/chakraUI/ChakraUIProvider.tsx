import { ChakraProvider } from '@chakra-ui/react';
import { ChakraUIProviderProps } from './type';

const ChakraUIProvider = ({ children }: ChakraUIProviderProps) => {
  return <ChakraProvider>{children}</ChakraProvider>;
};

export default ChakraUIProvider;
