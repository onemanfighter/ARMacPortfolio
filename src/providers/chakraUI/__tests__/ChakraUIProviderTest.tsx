import { render, screen } from '@testing-library/react';
import ChakraUIProvider from '../ChakraUIProvider';
import { Text } from '@chakra-ui/react';

describe('ChakraUIProvider', () => {
  it('should render correctly', () => {
    const { container } = render(
      <ChakraUIProvider>
        <Text>App</Text>
      </ChakraUIProvider>,
    );

    expect(container).toMatchSnapshot();
    expect(screen.getByText('App')).toBeDefined();
  });
});
