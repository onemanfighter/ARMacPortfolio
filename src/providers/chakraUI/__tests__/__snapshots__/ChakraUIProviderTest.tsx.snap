// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChakraUIProvider should render correctly 1`] = `
<div>
  <p
    class="chakra-text css-0"
  >
    App
  </p>
  <span
    hidden=""
    id="__chakra_env"
  />
</div>
`;
